src/main/java/org/spongepowered/common/mixin/core/entity/monster/CreeperEntityMixin.java

```diff
@@ -55,15 +57,15 @@
     @Shadow private int fuseTime;
     @Shadow private int explosionRadius;
 
-    @Shadow public abstract void ignite();
-    @Shadow public abstract int getCreeperState();
-    @Shadow public abstract void setCreeperState(int state);
-    @Shadow private void explode() { } // explode
+    @Shadow public abstract void shadow$ignite();
+    @Shadow public abstract int shadow$getCreeperState();
+    @Shadow public abstract void shadow$setCreeperState(int state);
+    @Shadow private void shadow$explode() { } // explode
 
-    private int fuseDuration = 30;
-    private boolean interactPrimeCancelled;
-    private boolean stateDirty;
-    private boolean detonationCancelled;
+    private int impl$fuseDuration = 30;
```

Comment:
The constant here should be shoved into the related (or new) `Constants` nested class.